autoload/rails.vim

```diff
@@ -1988,16 +1990,26 @@ function! rails#get_binding_for(pid) abort
     elseif executable('netstat')
       let output = system('netstat -antp')
       let binding = matchstr(output, '\S\+:\d\+\ze\s\+\S\+\s\+LISTEN\s\+'.a:pid.'/')
-      return empty(binding) ? '' : 'http://' . s:sub(binding, '^([^[]*:.*):', '[\1]:')
+      if empty(binding)
+        return ''
+      endif
```

Comment:
Can be dropped; we already guard against this after the outer `endif`.

Reply:
Pushing it now.  Thanks for catching that.